frame/sudo/src/lib.rs

```diff
@@ -110,6 +110,11 @@ mod mock;
 #[cfg(test)]
 mod tests;
 
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+pub mod weights;
+pub use weights::*;
```

Comment:
```suggestion
```

I dont think we normally do this.

Reply:
yeah, noticed new weights are generated for polkadot, so no need to export.

Reply:
After investigating, it is a standard approach for including the WeightInfo for use in the greater pallet call macro.